docs/helpers/env_vars.yaml

```diff
@@ -8024,21 +8229,21 @@ OCIS_EVENTS_ENABLE_TLS:
   removalVersion: ""
   deprecationInfo: ""
 OCIS_EVENTS_ENDPOINT:
-  name: OCIS_EVENTS_ENDPOINT;FRONTEND_EVENTS_ENDPOINT
+  name: OCIS_EVENTS_ENDPOINT;SEARCH_EVENTS_ENDPOINT
   defaultValue: 127.0.0.1:9233
   type: string
   description: The address of the event system. The event system is the message queuing
     service. It is used as message broker for the microservice architecture.
-  introductionVersion: "5.0"
+  introductionVersion: pre5.0
```

Comment:
I wonder why this is happening - how does the change to `SEARCH_EVENTS_ENDPOINT` mean that this entry is now introduced `pre5.0` when previously it said that it was introduced in `5.0`